src/components/Header/index.tsx

```diff
@@ -0,0 +1,252 @@
+				</Link>
+			</NavigationItem>
+		);
+	});
+};
+
+export const Wrapper = styled.div`
+	position: relative;
+	width: 100%;
+	background: white;
+	z-index: 5;
+
+	+ div:first-of-type {
+		margin-top: 80px;
+	}
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		box-shadow: 0px 0px 21px 5px rgba(152, 173, 227, 0.2);
+	}
+`;
+
+const Actions = styled(Box)`
+	justify-content: flex-end;
+	align-items: center;
+`;
+
+const MobileActions = styled(Flex)`
+	display: none;
+	min-height: 84px;
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		display: flex;
+	}
+`;
+
+const NavigationItems = styled(Flex)<{ menuOpen?: boolean }>`
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		top: 0;
+		right: 0;
+		width: 300px;
+		bottom: 0;
+		justify-content: flex-start;
+		transform: translateX(100%);
+		transition: transform 200ms ease-in-out;
+		padding: 16px;
+		overflow: auto;
+		flex-wrap: nowrap;
+
+		${(props) =>
+			props.menuOpen
+				? css`
+						position: fixed;
+						transform: translateX(0%);
+				  `
+				: ''}
+	}
+`;
+
+export const Navbar = styled(Flex)`
+	min-height: 80px;
+	position: relative;
+	flex-direction: column;
+	justify-content: center;
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		min-height: auto;
+		padding-top: 16px;
+		padding-bottom: 16px;
+	}
+`;
+
+export const Header = (props: HeaderProps) => {
+	const [mobileMenuOpen, setMobileMenuOpen] = React.useState(false);
+
+	const closeMenu = () => {
+		setMobileMenuOpen(false);
+	};
+	const openMenu = () => {
+		setMobileMenuOpen(true);
+	};
+	return (
+		<Wrapper>
+			<Container px={[3, 3, 3, 0]}>
+				<Navbar>
+					<Flex justifyContent="space-between" alignItems="center" width="100%">
+						<Flex alignItems="center">
+							<Link href={props.rootRoute} style={{ lineHeight: 1 }}>
+								<Img alt={props.logoAltText} src={props.logo} height="42" />
+							</Link>
+						</Flex>
+						{mobileMenuOpen && (
+							<Fixed
+								top
+								left
+								bottom
+								display={['block', 'block', 'none', 'none']}
+								right
+								bg="rgba(0, 0, 0, 0.25)"
+								onClick={closeMenu}
+							/>
+						)}
+						<Flex>
+							<NavigationItems
+								menuOpen={mobileMenuOpen}
+								flexDirection={['column', 'column', 'row', 'row']}
+								flexWrap="wrap"
+								alignItems={['flex-start', 'flex-start', 'center', 'center']}
+							>
+								{mobileMenuOpen && (
+									<Flex
+										display={['block', 'block', 'none', 'none']}
+										justifyContent="flex-end"
+									>
+										<Button
+											plain
+											underline={false}
+											p="5px"
+											mt="-8px"
```

Comment:
I am not a fan of using negative margins if not truly necessary, and this feels like it can be avoided (it seems it looks nice without the margins)